portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/displaypagetemplate/DisplayPageTemplatesWithItemPreview.testcase

```diff
@@ -480,4 +480,116 @@ definition {
+            	vocabularyName = "Audience");
+
+        	JSONCategory.addCategory(
+             	groupName = "Global",
+             	title = "Stage Category Name",
+             	vocabularyName = "Stage");
+        }
+
+        task ("Add a display page template for Category") {
+        	JSONLayoutpagetemplate.addDisplayPageTemplateEntry(
+           	 	contentType = "Category",
+          	    displayPageTemplateEntryName = "Display Page Name",
+            	groupName = "Test Site Name");
+        }
+
+        task ("Add a Button fragment to display Page Template") {
+            DisplayPageTemplatesAdmin.openDisplayPagesAdmin(siteURLKey = "test-site-name");
+
+            DisplayPageTemplatesAdmin.gotoDisplayPage(displayPageName = "Display Page Name");
+
+            PageEditor.addFragment(
+             	collectionName = "Basic Components",
+             	fragmentName = "Button");
+        }
+
+        task ("Mapping the Name field to Heading fragment") {
+         	PageEditorMapping.mapEditableTextToField(
+             	field = "Vocabulary",
+             	fragmentName = "Button",
+            	id = "link");
+        }
+  
+		task ("Select the category via preview item selector") {
+			WaitForLiferayEvent.initializeLiferayEventLog();
+
+			Click(locator1 = "ControlMenu#DISPLAY_PAGE_PREVIEW_ITEM_SELECTOR");
+
+			DropdownMenuItem.click(menuItem = "Select Other Item");
+
+			SelectFrame(locator1 = "IFrame#MODAL_BODY");
+
+			Portlet.expandTree();
+		}
+
+		task ("Assert the category of Topic") {
+			AssertElementPresent(
+				key_nodeName = "Topic Category Name",
+				locator1 = "Treeview#NODE_UNSELECTED");	
+		}
+
+		task ("Assert Audience and Stage Vocubalaries are not present") {
+			AssertElementNotPresent(
+				key_nodeName = "Audience (Global)",
+				locator1 = "Treeview#NODE_UNSELECTED");
+
+			AssertElementNotPresent(
+				key_nodeName = "Stage (Global)",
+				locator1 = "Treeview#NODE_UNSELECTED");
+
+			SelectFrameTop();
+		}
+
+		task ("Close the iframe") {
+			IFrame.closeFrame();
+		}
+
+		task ("Open the Preview modal") {
+			WaitForLiferayEvent.initializeLiferayEventLog();
+
+			Click(locator1 = "PageEditor#PREVIEW");
+		}
+
+		task ("Select the category in model") {
+			Click(locator1 = "Modal#DISPLAY_PAGE_PREVIEW_ITEM_SELECTOR");
+
+			DropdownMenuItem.click(menuItem = "Select Other Item");
+
+			SelectFrame(
+				key_title = "Select",
+				locator1 = "IFrame#MODAL_ANY");
+
+			Portlet.expandTree();
+		}
+
+		task ("Assert the category of Topic in modal") {
+			AssertElementPresent(
+				key_nodeName = "Topic Category Name",
+				locator1 = "Treeview#NODE_UNSELECTED");	
+		}
+
+		task ("Assert Audience and Stage Vocubalaries are not present in modal") {
+			AssertElementNotPresent(
+				key_nodeName = "Audience (Global)",
+				locator1 = "Treeview#NODE_UNSELECTED");
+
+			AssertElementNotPresent(
+				key_nodeName = "Stage (Global)",
+				locator1 = "Treeview#NODE_UNSELECTED");
+		}
```

Comment:
Add extra steps to clear global categories in this case. If the case is running on localhost, the dirty data probably affect the following cases.